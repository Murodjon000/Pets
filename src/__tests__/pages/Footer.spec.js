import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../../Components/Footer';

test('should correctly render Footer page', () => {
  const component = renderer.create(<Footer />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
