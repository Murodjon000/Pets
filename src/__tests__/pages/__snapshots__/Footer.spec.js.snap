// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render Footer page 1`] = `
<footer
  className="footer__wrapper"
>
  <a
    aria-current="page"
    href="/"
    onClick={[Function]}
  >
    PetFinder
  </a>
</footer>
`;
